binstar_client/utils/pprint.py

```diff
@@ -42,17 +42,17 @@ def pprint_package(package, access=True, full_name=True, revision=False):
 
     package['access'] = 'published' if package.get('published') else 'public' if package['public'] else 'private'
 
-    if 'conda_platforms' in package:
-        package['conda_platforms'] = ', '.join(package['conda_platforms'])
+    if package.get('conda_platforms'):
+        package['conda_platforms'] = ', '.join([str(x) for x in package['conda_platforms'] if x is not None])
```

Comment:
Isn't it the same to just do

```
package['conda_platforms'] = ', '.join([str(x) for x in package['conda_platforms'] if x])
``` 
?

Reply:
Well, x could evaluate False for other reasons than being None. 

```python
", ".join(str(x) for x in [0,1,2,3] if x)
```

Filters out the 0.

You're right about the generator though.

Reply:
Looks good, yes, it is better to use `is not None` in this case :-)